htcs-vehicle/src/state.c

```diff
@@ -22,7 +22,7 @@ void adjustState(State* state, unsigned elapsedMs) {
         if (state->lane == EXPRESS_LANE && state->speed > state->attributes.maxSpeed) {
             state->speed = state->attributes.maxSpeed;
             state->accelerationState = MAINTAINING_SPEED;
-        } else if (state->lane != EXPRESS_LANE && state->speed > state->attributes.preferredSpeed) {
+        } else if (state->speed > state->attributes.preferredSpeed) {
             state->speed = state->attributes.preferredSpeed;
```

Comment:
Bocsi, ezt én ninja commitoltam masterbe, de úgy látszik felülírtad :) szerintem kelleni fog ide ez a
lane != EXPRESS_LANE

Reply:
iigen, ezt benéztem, de utánna a be "cherry-pick"eltem, tehát a második commitban elvileg benne van